x-pack/plugins/aiops/public/components/change_point_detection/charts_grid.tsx

```diff
@@ -76,7 +85,7 @@ export const ChartsGrid: FC<{
           v.timestamp
         }_${v.p_value}`;
         return (
-          <EuiFlexItem key={key}>
+          <EuiFlexItem key={key} style={style}>
```

Comment:
1. I find it misleading that `style` is applied to each child element instead of the root. 
2. If style customization is needed, it'd be better to use styled components approach. We use [emotion](https://emotion.sh/docs/introduction) for such cases. 
3. As I mentioned in the other comment, I reckon it'd be better to pass the `gutterSize` property for the `EuiGrid` in this case.